pages/dkp/kommander/2.2/DKP-insight/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: DKP Insight
+title: DKP Insight
+excerpt: Use DKP insight to monitor and tune your enviornment
+menuWeight: 55
+beta: true
+enterprise: false
+---
+
+<p class="message--note"><strong>NOTE:</strong> Insights capability is offered with this release as a Technical preview only. Future enhancements include add-on licensing, a seamless user experience, and additional insight items.</p>
+
+DKP Insights is D2iQ’s latest analytics tool, designed to assist Kubernetes Administrators with daily tasks, such as making sure clusters are secure, debugging, monitoring, and anticipating future issues with your cluster. Insights provides you predictive analytics capability that provides greater support productivity, speed, and reduced cost. An Insights module, integrated into the DKP, collects configuration, logs, events, and metrics from DKP deployments and generates predictive insights on potential issues of varying critical levels so that you can quickly identify and resolve issues that would save a lot of time and money by not escalating to D2IQ's support.
+
+Insights runs as part of DKP within your environment, so it works seamlessly within networked or air-gapped environments. Insights collects data from multiple points, such as metrics, logs, and events, and then uses a set of heuristics and models to interpret the data and let you know what’s going on. Events display in a user-friendly, interactive table, showing critical issues, warnings, and general notices you need to ensure your environment is running smoothly. 
+
+You can filter the Insights summary cards using one or more categories for a selected cluster and project:
```

Comment:
Currently, users can only select one category at a time, so it might be best to not include "one or more".

Reply:
1. Can we refine this "Insights runs as part of DKP within your environment, so it works seamlessly within networked or air-gapped environments". for example, we can use something like "The insights capability in DKP works seamlessly within networked or air-gapped environments. It collects metrics, logs, and events from multiple data sources and uses rule-based heuristics to detect issues from the data and generate insight items for the users.
2.  Didnt quite understand what you want to convey here: "Events display in a user-friendly, interactive table, showing critical issues, warnings, and general notices you need to ensure your environment is running smoothly. "
3. There are no summary cards now (You can filter the Insights summary cards). We use insights items in a table that can be filtered